src/pagination/__tests__/__snapshots__/pagination.spec.ts.snap

```diff
@@ -121,7 +122,8 @@ exports[`Pagination renders markup without state 1`] = `
   <ais-instantsearch>
     <ais-pagination>
       <div
-        class="ais-Pagination"
+        class="ais-Pagination ais-Pagination--noRefinement"
+        ng-reflect-ng-class="ais-Pagination,ais-Pagination-"
```

Comment:
is this actually in the Dom?

Reply:
No, that's for debugging purposes: https://stackoverflow.com/a/43473682/9940315.